File: app/src/main/java/com/maximus/productivityappfinalproject/domain/SetClosestTimeUseCase.java
package com.maximus.productivityappfinalproject.domain;

import com.maximus.productivityappfinalproject.data.AppsRepository;

public class SetClosestTimeUseCase {
    private AppsRepository mAppsRepository;


    public SetClosestTimeUseCase(AppsRepository appsRepository) {
        mAppsRepository = appsRepository;
    }

    public void setClosestHour(long hour) {
        mAppsRepository.setClosestHour(hour);
    }

    public void setClosestDay(long day) {
        mAppsRepository.setClosestDay(day);
    }
}

File: app/src/main/java/com/maximus/productivityappfinalproject/domain/SetClosestTimeUseCase.kt
package com.maximus.productivityappfinalproject.domain

import com.maximus.productivityappfinalproject.data.AppsRepository
import javax.inject.Inject

class SetClosestTimeUseCase @Inject constructor(private var appsRepository: AppsRepository){

    fun setClosestHour(hour: Long) {
        appsRepository.closestHour = hour
    }

    fun setClosestDay(day: Long) {
        appsRepository.closestDay = day
    }

}